schemas/panels/time-series/mig.cuepart

```diff
@@ -21,6 +24,16 @@ if #panel.type == "timeseries" || #panel.type == "graph" {
                     if #panel.legend.alignAsTable != _|_ if #panel.legend.alignAsTable { "Table" },
                     { "List" }
                 ][0]
+                values: [ for oldCalc, newCalc in #mapping.calc  
+                    // Check if the mapping field is set on the legend.
+                    if (#panel.legend[oldCalc] != _|_)  
+                        // Check if the mapping field is true. This has to be
+                        // done inside the prior check to avoid cue throwing
+                        // undefined errors when the value is not present.
```

Comment:
I *really* wish cue was smart enough to fail after the first conditional so I could do a single `if` with "and" logic, but sadly, it does not.

Reply:
@AntoineThebaud can we get some help with a few CUE questions, Julie put another one in Matrix as well

It’s probably fine to do the migration updates later, especially if it isn’t a 1:1 mapping like most of the examples in other cuepart files

Reply:
I _think_ I got everything working although it took me forever to figure it out because I don't know cue very well. It'll be good to get a look from Antoine as a reviewer to make sure I got it right though!

The comment about the conditionals was mostly context for reviewers (and a little bit of me whining) because this works differently than a lot of other languages. I think that's the best way to do it though based on looking at some [discussions](https://github.com/cue-lang/cue/discussions/2061#discussioncomment-4044662) in the cue github.